cypress/integration/CheckboxGroup/Displays_error.feature

```diff
@@ -0,0 +1,6 @@
+Feature: The CheckboxGroup field displays an error when invalid
+
+    Scenario: Form is submitted with none of the options selected
+        Given a CheckboxGroup with no selected value
```

Comment:
Shouldn't this only display an error if it's required?

Reply:
Yeah it actually does that:
```javascript
cy.visitStory('Testing:CheckboxGroup', 'Required')
```
But the feature texts don't reflect that. I'll amend this